Gemfile

```diff
@@ -1,4 +1,3 @@
 source 'https://rubygems.org'
-ruby '2.2.3'
```

Comment:
'Need' may be strong, but specifying a Ruby version in the Gemfile generally makes deployment more reliable. By making the bundle fail when running on an older (or different) Ruby version, you guard against changes in Ruby having unexpected effects on your codebase. (The Ruby ground shifting under your feet, as it were.)

Similarly, by knowing if and when the Ruby version _has_ changed on you, as was the case here, this deployment failure is a prompt to evaluate your codebase for any necessary changes.

If none of the other Federalist sites do that, I suggest we merge this as-is and take that question — should Federalist sites specify a Ruby version? — up separately. It may also be relevant to the cloud.gov team's plans for notifying app owners when cloud.gov's buildpacks change.